src/users/users.service.ts

```diff
@@ -107,4 +107,102 @@ export class UsersService {
       .where('userId = :userId', { userId })
       .execute();
   }
+
+  async getStudentCv(id: string): Promise<StudentCvResponse> {
+    const student = await this.studentProfileRepository.findOne({
+      select: [
+        'id',
+        'firstName',
+        'lastName',
+        'bio',
+        'githubUsername',
+        'courseCompletion',
+        'courseEngagement',
+        'projectDegree',
+        'teamProjectDegree',
+        'projectUrls',
+        'portfolioUrls',
+        'bonusProjectUrls',
+        'expectedTypeWork',
+        'targetWorkCity',
+        'expectedContractType',
+        'expectedSalary',
+        'canTakeApprenticeship',
+        'monthsOfCommercialExp',
+        'education',
+        'workExperience',
+      ],
+      where: {
+        id,
+      },
+    });
+
+    if (student) {
+      return student;
+    } else {
+      throw new HttpException(
+        'Student with this id does not exist',
```

Comment:
```suggestion
        'Nie znaleziono takiego studenta',
```
chyba apka ma być po polsku więc zwracajmy po polsku, tak wiem ja też chyba pisze po angielsku, ale no łatwiej dla frontu będzie wyświetlić message